src/ioctls/vcpu.rs

```diff
@@ -2826,4 +2883,199 @@ mod tests {
         }
         assert!(vcpu.vcpu_init(&kvi).is_ok());
     }
+
+    #[test]
+    #[cfg(target_arch = "x86_64")]
+    fn test_coalesced_pio() {
+        use crate::IoEventAddress;
+        use std::io::Write;
+
+        const PORT: u64 = 0x2c;
+        const DATA: u64 = 0x39;
+        const SIZE: u32 = 1;
+
+        #[rustfmt::skip]
+        let code = [
+            0xe6, 0x2c,   // out 0x2c, al
+            0xf4,         // hlt
+            0xe6, 0x2c,   // out 0x2c, al
+            0xf4,         // hlt
+        ];
```

Comment:
Can we use constants for the mnemonics?

E.g.
```suggestion
        use iced_x86::Mnemonic::{Hlt,Out};
        
        #[rustfmt::skip]
        let code = [
            Out as u8, 0x2c,
            Hlt as u8,
            Out as u8, 0x2c,
            Hlt as u8,
        ];
```